misc/chip_data.json

```diff
@@ -725,7 +725,7 @@
     },
     "84646ed2-015e-4a8b-9d37-5115cb9ebadc": {
       "ReadonlyName": "Atan",
-      "Description": "",
+      "Description": "This node allows you to use the math formula Atan",
```

Comment:
let's use the word "chip" instead of node where possible. we know people will call things nodes but we want to be consistent about official terminology